documentation/smg/06_deployment_process.md

```diff
@@ -0,0 +1,79 @@
+
+After ensuring that all tests are passing on the production branches, the developer should leave 
+the horace/herbert code tree and move his Matlab working folder to a location, where the Horace 
+distributive should be located. After that, he runs the `make_horace_deployment_kit` script, 
+which produces 5 archives. The main one -- ***horace_distribution_kit.zip*** containing the whole Horace
+code with demo files and tests and 4 reduced archives, the most important one is the 
+small but self-consistent ***Horace&Herbert_NoDemoNoTests.zip*** which contains 
+complete working code but without tests and demo files. 
+
+Then the developer should log into **shadow.isis.cclrc.ac.uk** and download these files to 
+**/isis/www/horace/kits** folder. This folder is available for downloading from the
+web at the location specified on 
+[Horace Distribution page.](http://horace.isis.rl.ac.uk/Download_and_setup)
+
+At the moment only Alex Buts (wkc26243) have write access to this folder. The developers who
+need write access to this folder should request it from 
+Freddie Akeroyd: <freddie.akeroyd@stfc.ac.uk>
+
+The [`make_horace_deployment_kit`](https://github.com/pace-neutrons/Horace/blob/master/admin/make_horace_deployment_kit.m)
+script performs the following operations:
+
+ - It runs [update_svn_revision_info Matlab script](https://github.com/pace-neutrons/Herbert/blob/master/admin/update_svn_revision_info.m) 
+    which updates svn revision information in all code files 
+    where this info is present, if the key `-update_version` is provided as input. 
+   (can be reduced to just `-u`)
+ - It runs [`make_horace_distribution_kit`](https://github.com/pace-neutrons/Horace/blob/master/admin/make_horace_distribution_kit.m)
+   with different options. This script
+   defines the code-tree folders, which should be distributed to users and copies this 
+   folders into the target location. The different script options define different amount and type
+   of the code to copy to obtain different archives above.
+ - Depending on input option, `make_horace_distribution_kit` may deploy 
+   [`make_herbert_distribution_kit`](https://github.com/pace-neutrons/Herbert/blob/master/admin/make_herbert_distribution_kit.m)
+   script, which does similar operation with Herbert code. 
+ - After all necessary code is copied into the target location, the script
+   p-codes the specified folders on request, compresses 
+   the files together with their location within the folders tree, 
+   and removes target location after compression. 
+
+## *production* branch channel.
+
+After completing the generation of zip code, developer merges the production branch to master and
+commits the changes to git.  This brings the updated code to all developers. 
+
+To provide the updated code to the ISIS users, the developer should update the production branch on 
+***isisompute*** or ***iDaaaS*** machines. The distributive is physically located on shared parallel file 
```

Comment:
"on isiscompute or iDaaaS" -- does it need to be updated on both, or are changes on one mirrored to the other sysdtem?

also there's a typo in "isisompute" here.